assets/queries/ansible/aws/db_instance_publicly_accessible/query.rego

```diff
@@ -0,0 +1,59 @@
+package Cx
+
+CxPolicy[result] {
+    document := input.document[i]
+    tasks := getTasks(document)
+    task := tasks[t]
+    isAnsibleTrue(task["community.aws.rds_instance"].publicly_accessible)
+    result := {
+        "documentId": document.id,
+        "searchKey": sprintf("name=%s.{{community.aws.rds_instance}}.publicly_accessible", [task.name]),
+        "issueType": "IncorrectValue",
+        "keyExpectedValue": "community.aws.rds_instance.publicly_accessible should be false",
+        "keyActualValue": "community.aws.rds_instance.publicly_accessible is true"
```

Comment:
Should specify the resource name to help pinpoint the failing resource

Reply:
we can specify only the task name, not the resource name, but ok. I'll put in the task name